styleguide/source/assets/js/webforms.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @file
+ * Contact us interactions.
+ *
+ * JavaScript should be made compatible with libraries other than jQuery by
+ * wrapping it with an "anonymous closure". See:
+ * - https://drupal.org/node/1446420
+ * - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
+ */
+(function ($, Drupal) {
+
+  var verifyFields = function(form) {
+    console.log("hello");
+    var $sections = form.find('section');
+    var $inputs = $('.webform-submission-form section *').filter(':input');
+    var $iconElement = $('.ama__form-steps__icon');
+    var errorSections = [];
+    var errorMessage = $('.form-item--error-message');
```

Comment:
Are you using the $ = jQuery object? If so, I like that pattern and we should do this for `errorMessage` and others. Give it another check.

Reply:
Yes, I am. I actually didn't need that variable. Good Catch!